### PR TITLE
NEP-6: Basic Token Standard (NRC-6)

### PR DESCRIPTION
**Pull request by LiuChaooo**
## Checklist:

- [x] I have read the **NEP Guideline** document.
- [x] I have checked my NEP Draft is ready.

### CODE REVIEW

**Comment by arisac** [comment]
Review Checklist for NEP Editors

`this checklist is for new NEP Draft`

**Check pr NEPs repo**
- [ ] new NEP is placed in /NEPS/nep-title.md
- [ ] assets is placed in /NEPS/assets/NEP-N/asset-name.ext
- [ ] no modifications to other files

**Check NEP formats**
- [ ] header
- [ ] all contents in English

Ask the applicant to make modifications to meet criterions above.

**Assign NEP Number**
- [ ] use pr number if no related issue had opened before
- [ ] change Title and NEP number in document
- [ ] rename nep-title.md to nep-number.md
- [ ] rename assets directory NEP-N to nep-number

**Arrange NEP Board Review**
tbd

**Comment by liuyong5653** [comment]
NEP Board Review

File: nep-6.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+## Specification
+
+### Meta Data
+
+| Item | Description | Behaviors/Properties |
+|:-|:-|:-|
+| name | Name of token | can not be changed |
+| symbol | Symbol of token | can not be changed |
+| decimals | decimals of Token | can not be changed |
```
Comment (VieYang) [inline nep-6.md:35]: name,symbol,decimals虽然用户不一定会改，但是标准中是不是不一定要“can not be changed”？

File: nep-6.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+| totalSupply | Total supply of Token |  |
+
+
+### Interaction / Functions
+
+| Function | Description | Behaviors/Properties |
+|:-|:-|:-|
+| transfer(recipient, amount) | Moves `amount` tokens from the caller's account to `recipient` |  |
+| allowance(owner, spender) | Returns the remaining number of tokens that `spender` is allowed to spend from `owner` |  |
```
Comment (VieYang) [inline nep-6.md:44]: 单独列举了Query, 那么Function这里是特指更改链的操作嘛？allowance(owner, spender) 是查询Query操作。

File: nep-6.md (hunk 1 above)
Comment (LiuChaooo) [reply nep-6.md:35]: name,symbol,decimals 要预留修改的空间吗？

File: nep-6.md (hunk 2 above)
Comment (LiuChaooo) [reply nep-6.md:44]: function 没有特指， query 特指查询函数

File: nep-6.md (hunk 1 above)
Comment (LiuChaooo) [reply nep-6.md]: 已修改，去掉这三个 can not be changed

File: nep-6.md (hunk 2 above)
Comment (LiuChaooo) [reply nep-6.md]: 已修改，allowance 归类到 query

File: NEPS/nep-6.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+# NEP 6: Basic Token Standard
```
Comment (evanliuchina) [inline NEPS/nep-6.md:1]: # NEP-6: Basic Token Standard (NRC-6)

File: NEPS/nep-6.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+| Item | Description |
+|:-|:-|
+| NEP | 6 |
+| Title | Basic Token Standard |
+| Author | [Yong Liu](mailto:liuyong5653@163.com), [Qu Jianwei](https://github.com/i29), [LiuChao](https://github.com/LiuChaooo) |
+| Discussions to | https://github.com/newtonproject/NEPs/issues/6 |
+| Status | Draft |
+| Type | Standard |
```
Comment (evanliuchina) [inline NEPS/nep-6.md:10]: Type: NRC

File: NEPS/nep-6.md (hunk 3 above)
Comment (liuyong5653) [reply NEPS/nep-6.md]: done

File: NEPS/nep-6.md (hunk 4 above)
Comment (liuyong5653) [reply NEPS/nep-6.md]: done

**Comment by liuyong5653** [comment]
NEP Board Review 4758691

**Pull request merged**